packages/cli/internal/pkg/cli/clierror/cli_error.go

```diff
@@ -24,19 +30,23 @@ func (e *Error) Unwrap() error {
 
 // New constructs an error message intended to be read by the CLI user. Holds context in the form of the invoked command (command),
 // the variables of the command (commandVars), the causal error (cause), and any suggested action the user might take to correct the problem (suggestedAction).
-func New(command string, commandVars interface{}, cause error, suggestedAction string) *Error {
-	log.Err(cause).Send()
-
-	actionableError, ok := cause.(ActionableError)
+func New(command string, commandVars interface{}, err error) *Error {
+	actionableError, ok := err.(*actionable.Error)
```

Comment:
I think it is better to use `errors.As` operation. 

For more information on error handling in Go 1.13+ please read:
- https://go.dev/blog/go1.13-errors